src/css/application.css

```diff
@@ -396,21 +396,21 @@ body {
 
 .editors__editor-container {
   box-sizing: border-box;
-  flex: 0 1 100vh;
   border-right: 4px solid var(--color-dark-gray);
-  border-bottom: 4px solid var(--color-dark-gray);
   position: relative;
   z-index: 0;
   display: flex;
-}
-
-.editors__editor-container:last-child {
-  border-bottom: 0;
+  flex-direction: column;
```

Comment:
I take that back. It's not technically necessary, but without it the `Editor` is properly sized because it's flexing horizontally, which feels a little weird.